crates/notion-core/src/tool.rs

```diff
@@ -420,9 +420,10 @@ impl Tool for Npx {
         if let Some(ref platform) = session.current_platform()? {
             let image = platform.checkout(session)?;
 
-            // npx was only included with Node >= 8.2.0. If less than that, we should include a helpful error message
-            let required_node = VersionSpec::parse_requirements(">= 5.2.0")?;
-            if required_node.matches(&image.node.npm) {
+            // npx was only included with Node >= 8.2.0 which shipped with npm 5.2.0. If the npm
```

Comment:
Super minor, but I would get rid of the mention of `Node` at all, since we're directly comparing `npm` versions. Something like:

```
npx was only included with npm 5.2.0 and above. If the npm version
is less than that, we should include a helpful error message
```

Reply:
Good point, updated!